src/legacy/core_plugins/kibana/public/table_list_view/table_list_view.js

```diff
@@ -351,7 +351,10 @@ class TableListViewUi extends React.Component {
           defaultMessage: 'Actions'
         }),
         width: '100px',
-        actions
+        actions,
+        mobileOptions: {
```

Comment:
Yeah I forgot that this won't work because the Actions column is different from regular table cells and doesn't currently support this object. I need to think on the best way to handle this because should we be hiding controls just based on window size? Maybe/maybe not? We might want to consider other factors too.

Reply:
I think we'll still hide controls in case the user doesn't have write permission, if you mean that? But in this case I think we just need to hide whenever we don't have the grid view (to actually place panels and give them their coordinates) available.